framework/src/mesh/MooseMesh.C

```diff
@@ -500,21 +500,23 @@ MooseMesh::cacheChangedLists()
 }
 
 ConstElemPointerRange *
-MooseMesh::refinedElementRange()
+MooseMesh::refinedElementRange() const
```

Comment:
Doesn't break any apps but those ranges are used throughout the various threading loops. We'll have to update all those to take const versions. This is already a large enough PR.